src/CodeReport/CMakeLists.txt

```diff
@@ -5,9 +5,13 @@
 cmake_minimum_required(VERSION 3.15)
 
 project(CodeReport)
-add_library(CodeReport INTERFACE)
+add_library(CodeReport OBJECT)
```

Comment:
It's the first time I see `OBJECT` used in Orbit. Do we prefer this to `STATIC` or what's the reason?

Reply:
Yes, we prefer OBJECT :slightly_smiling_face:  An OBJECT library is just a bunch of .o-files, while a static library is always linked into a .a/.lib-file. Since we don't ship the the static libraries, it's just a waste of time and disk space to generate them.

At some point I will move everything to OBJECT targets.

Reply:
Hehe I was always wondering whether all those generated static libraries were really necessary. Thanks!

Reply:
Feel free to try out OBJECT next time. :smile: